.github/workflows/build.yaml

```diff
@@ -18,15 +18,7 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-
-      - name: pub get
-        run: pub get
-
-      - name: dart format
-        run: dart format --output=none --set-exit-if-changed .
-
-      - name: dart analyze
-        run: dart analyze
-
-      - name: dart test
-        run: dart test
+      - run: dart pub get
+      - run: dart format --output=none --set-exit-if-changed .
```

Comment:
We might want a shorthand for `dart format --output=none --set-exit-if-changed`; it seems like a common set of flags for a CI like task to want.

Reply:
I agree! Like `dart format verify` or similar...